conans/test/functional/command/remote_test.py

```diff
@@ -3,6 +3,7 @@
 import unittest
 from collections import OrderedDict
 
+from conans.test.utils.conanfile import TestConanFile
```

Comment:
New import, but it is not used in the code added.

Besides that, the PR is ready to merge IMO